lib/src/command_runner.dart

```diff
@@ -85,8 +92,29 @@ class VeryGoodCommandRunner extends CommandRunner<int> {
 
   @override
   Future<int?> runCommand(ArgResults topLevelResults) async {
+    final isUpToDate = await _pubUpdater.isUpToDate(
+      packageName: 'very_good_cli',
```

Comment:
can we define a constant for the `packageName` so that we don't have to repeat it in multiple places?